docs/USING_BULK_SELECTION.md

```diff
@@ -0,0 +1,161 @@
+});
+```
+
+Selection state is an [Immutablejs Ordered Map](https://facebook.github.io/immutable-js/docs/#/OrderedMap)
+
+```
+const getSelectedIds = ()=> {
+
+   const bulkMap = selection.get("bulk"); // selection is React-Redux-Grid reducer available from the store
+   const selectedIds = bulkMap.get("indexes");
+
+   if (undefined !== selectedIds) {
+       return selectedIds; 
+    } else {
+        // you could dispatch a notification... 
+        // this.props.displayWarningRequestMessage("No rows selected");
+        console.warn("BulkPage selectedIds were false!");
+        return [];
+    }  
+};
+```
+
+If you set activeCls to true, the rows will change color as you hover over and click on them and the checkbox will also toggle state.
+
+```
+SELECTION_MODEL: {
+        mode: 'checkbox-multi',
+        enabled: true,
+        allowDeselect: true,
+        activeCls: 'active',
+        selectionEvent: 'singleclick',
+        tabIndex: "-1"
+    },
+```
+
+By default, the checkbox has tab-index of "-1" so that when you tabbing through the cells, it skips the checbox allowing you to focus on one column.  
+
+If you set activeCls to false, the rows will change color as you hover over and click on them but the checkbox will not toggle state until you actually click on the checkox. With selectionEvent set to doubleclick, this doesn't mean you need to click the checkbox twice quickly, it feels like you are clicking it normally. 
+
+
+
+```
+SELECTION_MODEL: {
+                mode: 'checkbox-multi',
+                enabled: true,
+                allowDeselect: true,
+                activeCls: 'false',
+                selectionEvent: false
+            },
+```   
+
+
+```
+SELECTION_MODEL: {
+                mode: 'checkbox-multi',
+                enabled: true,
+                allowDeselect: true,
+                activeCls: 'false',
+                selectionEvent: 'doubleclick'
+            },
+```     
+
+
+
+
+#### Bulk & Selection Plugins 
+
+```
+export const plugins = {
+    SELECTION_MODEL: {
+        mode: 'checkbox-multi',
+        enabled: true,
+        allowDeselect: true,
+        activeCls: 'active', // the css for the active cell 
+        selectionEvent: 'singleclick' // singleclick vs doubleclick 
+    },
+    BULK_ACTIONS: {
+        enabled: true,
+        actions: [
+            {
+                text: 'Move',
+                EVENT_HANDLER: () => {
+
+                }
+            },
+            {
+                text: 'Add',
+                EVENT_HANDLER: () => {
+
+                }
+            }
+        ]
+    }
+};
+```
+
+#### Selecting only the checkbox
+
+To select a row, simply click anywhere on that row, and this is the default behaviour. If you want to limit the selection to the checkbox, you will have to alter the css.  
+
+You can also call this function componentDidMount or componentDidUpdate
+
+const columnIndex = 0; 
+updateTabIndex(columnIndex)
+
+#### Disabling Tab Index
```

Comment:
As for this, what I really wanted was to be able to configure the column and pass in tabIndex prop for each one. If I want to disable it, I could pass "-1"